local-cluster/exe/seed-network.hs

```diff
@@ -0,0 +1,89 @@
+{-# LANGUAGE TypeApplications #-}
+
+import CardanoClient (
+  buildAddress,
+  postSeedPayment,
+  queryProtocolParameters,
+  queryUtxo,
+ )
+import CardanoCluster (asSigningKey, availableInitialFunds, defaultNetworkId)
+import Hydra.Chain.Direct.Util (readFileTextEnvelopeThrow)
+import Hydra.Ledger.Cardano (Lovelace, getVerificationKey)
+import Hydra.Prelude
+import Options.Applicative (
+  Parser,
+  ParserInfo,
+  auto,
+  execParser,
+  fullDesc,
+  header,
+  help,
+  helper,
+  info,
+  long,
+  metavar,
+  option,
+  progDesc,
+  showDefault,
+  strOption,
+  value,
+ )
+
+data Options = Options
+  { cardanoNodeSocket :: FilePath
+  , cardanoSigningKey :: FilePath
+  , amountLovelace :: Lovelace
+  }
+
+seedNetworkOptionsParser :: Parser Options
+seedNetworkOptionsParser =
+  Options
+    <$> strOption
+      ( long "cardano-node-socket"
+          <> metavar "FILE"
+          <> help "The path to the Cardano node domain socket for client communication."
+          <> value "node.socket"
+          <> showDefault
+      )
+    <*> strOption
+      ( long "cardano-signing-key"
+          <> metavar "FILE"
+          <> help "The path to the signing key file used for committing UTxO. This file used the same 'Envelope' format than cardano-cli."
+          <> value "me.sk"
+          <> showDefault
+      )
+    <*> ( fromIntegral @Integer
+            <$> option
+              auto
+              ( long "commit-amount"
+                  <> metavar "LOVELACE"
+                  <> help "The amount of Lovelaces in the generated UTXO used for committing."
+                  <> value 100_000_000
```

Comment:
I am not sure it's good to have a default here. This is certainly only use for devnet / testnet at the moment, but generally speaking, that's the kind of option I prefer to not have default for so users are 100% aware of how much they're committing.